### ISSUE TITLE
led storm rally 2011 graphics problem

### ISSUE THREAD

**Issue by kounabi22** [finalburnneo/FBNeo#1302]
Graphics problem apear in led storm rally 2011

**Comment by kounabi22** [comment]
![325425112_2406868766156457_7099511849647452580_n](https://user-images.githubusercontent.com/78277576/212433772-c72600ad-2c21-4f6a-b930-1c3c12de87a3.jpg)

**Comment by dinkc64** [comment]
I got to stage 4 and didn't see it so far...
where's the problem at?

**Comment by kounabi22** [comment]
begining of first stage.... i will try delete cfg files in fbneo to see if it fix it.

**Comment by kounabi22** [comment]
i download it again.its ok now.

**Comment by barbudreadmon** [comment]
This is happening because you overclocked the emulated cpu. 
For the reminder, overclocking the emulated cpu is not safe.

**Comment by kounabi22** [comment]
it must the option highscore enable......if i disable it automatic fix the gfx problem.

**Comment by barbudreadmon** [comment]
indeed, i mistakenly thought it was related to overclock because the issue happened right after i tried overclocking it, but actually it was happening because a hiscore file had been generated

**Comment by dinkc64** [comment]
hiscores have been fixed for this game, thanks for the bug report :)

**Comment by kounabi22** [comment]
thanks.
